hera_qm/tests/test_xrfi.py

```diff
@@ -464,5 +464,35 @@ def test_summarize_flags(self):
         self.assertEqual(data['version'], hera_qm_version_str)
 
 
+class TestBroadcast(unittest.TestCase):
+    def test_summarize_flags(self):
+        from pyuvdata import UVData
+
+        infile = os.path.join(DATA_PATH, 'zen.2457698.40355.xx.HH.uvcAA')
+        uv = UVData()
+        uv.read_miriad(infile)
+
+        # Test baseline thresholding
+        uv.flag_array[0, 0, uv.Nfreqs / 2, 0] = True
+        bflags = xrfi.broadcast_flags(uv, bl_threshold=0.)
+        nbl = np.sum(uv.time_array == uv.time_array[0])
+        self.assertEqual(bflags.sum(), nbl)
+        # Check thresholding works correctly
+        bflags = xrfi.broadcast_flags(uv, bl_threshold=0.5)
+        self.assertEqual(bflags.sum(), 1)
+
+        # Test frequency thresholding
+        t_ind = np.where(uv.time_array == uv.time_array[0])[0]
+        uv.flag_array[t_ind, 0, 0:(uv.Nfreqs * 3 / 4), 0] = True
```

Comment:
To future-proof ourselves, we could add `from __future__ import division` at the top of the script, and change this to invoke explicit integer division: `0:(uv.Nfreqs * 3 // 4)`. Or we could cast the result to `int`, either way.

Reply:
Fixed.